plugins/catalog/src/components/EntityPageOverview/EntityPageOverview.tsx

```diff
@@ -51,6 +52,16 @@ export const EntityPageOverview: FC<{ entity: Entity }> = ({ entity }) => {
             <GithubActionsWidget entity={entity} branch="master" />
           </Grid>
         )}
+        {entity.metadata?.annotations?.['backstage.io/sentry-project-id'] && (
```

Comment:
Happy to change the naming of the annotation to something more suitable if required. Is there a good place to document the annotations?

Reply:
I was under the impression this might be 'sentry.com/project-id'? I recall seeing the usage of 'github.com/project-slug' elsewhere.

Reply:
Maybe not exactly that though https://www.sentry.com/ 😅 

`sentry.io/project-id`, sentry.io

Reply:
@muffix are you up to changing it to that? ^ also below. Or to break it out into a separate variable `const sentryProjectId = entity.metadata?.annotations?.['sentry.io/project-id']` that may or may not be undefined, to avoid the repetition

Reply:
Hang on, just checking - the API lists this parameter as project slug, not id `https://docs.sentry.io/api/projects/get-project-details/`, should it really be named `sentry.io/project-slug` instead?

Reply:
Uh but then in other places they do use the opaque ID instead https://docs.sentry.io/api/pagination/ - are both supported?

Reply:
But I'm going to go with the hypothesis that slug is the better name, and even if both id and slug can be passed there, users will prefer to put the more usable/understandable slug in there anyway.